### PR TITLE
added the syntax examples for var,collection,str

### PR DESCRIPTION
**Pull request by Jaikishan0511**

### CODE REVIEW

File: JS/maps.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+var map = new Map();
+map.set("r1", "user").set("r2", "admin");
+/*console.log(map);*/
```
Comment (Shrikant9) [inline JS/maps.js]: Remove the commented code if not required.

File: JS/maps.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+var map = new Map();
+map.set("r1", "user").set("r2", "admin");
```
Comment (Shrikant9) [inline JS/maps.js:2]: `.set().set()` ?

Comment (Jaikishan0511) [reply JS/maps.js:2]: tried chaining

File: JS/maps.js (hunk 1 above)
Comment (Jaikishan0511) [reply JS/maps.js]: tried all the different methods for map

Comment (Jaikishan0511) [reply JS/maps.js]: only for reference, instead of creating no. of files I executed them one by one

**Review by Shrikant9 — APPROVED** [review]

**Pull request merged**